internal/vcs/create-git-repos.go

```diff
@@ -96,51 +124,42 @@ type organizationQueryResponse struct {
 	}
 }
 
-func createOrganizationOwnedRepository(moduleDir string, githubApiKey string, organizationName string) error {
-
-	fmt.Printf("Creating org owned repository for module: %s\n", moduleDir)
+// isOrganizationOwned will determine if ownerName is an organization.
+// If ownerName is an organization it's id will be returned.
+func isOrganizationOwned(ownerName string, githubApiKey string) (bool, string, error) {
+	oRequest := graphql.NewRequest(getOrganizationQuery)
+	oRequest.Var("ownerName", ownerName)
+	oRequest.Header.Add("Authorization", fmt.Sprintf("Bearer %s", githubApiKey))
 
-	// create client and organization query
-	var bearer = fmt.Sprintf("Bearer %s", githubApiKey)
-	client := graphql.NewClient("https://api.github.com/graphql")
-	orgIdReq := graphql.NewRequest(getOrganizationQuery)
-	orgIdReq.Var("organizationName", organizationName)
-	orgIdReq.Header.Add("Authorization", bearer)
-
-	var orgIdResp organizationQueryResponse
+	var oResponse organizationQueryResponse
+	c := graphql.NewClient("https://api.github.com/graphql")
 	ctx := context.Background()
-	if err := client.Run(ctx, orgIdReq, &orgIdResp); err != nil {
-		return err
-	}
-	organizationId := orgIdResp.Organization.Id
-
-	// create mutation and run it
-	req := graphql.NewRequest(createOrganizationRepositoryMutation)
-	req.Var("repoName", moduleDir)
-	req.Var("repoDescription", fmt.Sprintf("Repository for %s", moduleDir))
-	req.Var("ownerId", organizationId)
-	req.Header.Add("Authorization", bearer)
-	if err := client.Run(ctx, req, nil); err != nil {
-		return err
-	}
+	if err := c.Run(ctx, oRequest, &oResponse); err != nil {
 
-	fmt.Printf("Repository successfully created for module: %s\n", moduleDir)
+		notAnOrgMessage := fmt.Sprintf("graphql: Could not resolve to an Organization with the login of '%s'.", ownerName)
+		if err.Error() == notAnOrgMessage {
+			return false, "", nil
+		}
+		return false, "", err
+	}
+	organizationId := oResponse.Organization.Id
 
-	return nil
+	return true, organizationId, nil
 }
 
-type InitialCommands struct {
+type initialCommands struct {
 	description string
 	command     string
 	args        []string
 }
 
-// do initial commit to a repository
-func doInitialCommit(moduleDir string, remoteRepository string) error {
-	fmt.Printf("Initializing repository for module: %s\n", moduleDir)
+// doInitialCommit runs the git commands that initialize and do the first commit to a repository.
+func doInitialCommit(ownerName string, repositoryName string) error {
+	fmt.Printf("Initializing repository for module: %s\n", repositoryName)
 
-	remoteOrigin := fmt.Sprintf("%s/%s.git", remoteRepository, moduleDir)
-	commands := []InitialCommands{
+	remoteOrigin := fmt.Sprintf("git@github.com:%s/%s.git", ownerName, repositoryName)
+	fmt.Printf("remote origin: %s\n", remoteOrigin)
```

Comment:
Probably not necessary to print

Reply:
Actually I think all the status prints in this probably aren't very useful to the user. Just one per repo saying we are creating a repo should be enough.